Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -672,7 +676,7 @@ void startAppLinkFlow(FragmentActivity activity, VenmoIntentData input) throws J
                 .appendQueryParameter("braintree_access_token", input.getConfiguration().getVenmoAccessToken())
                 .appendQueryParameter("braintree_environment", input.getConfiguration().getVenmoEnvironment())
                 .appendQueryParameter("resource_id", input.getPaymentContextId())
-                .appendQueryParameter("braintree_sdk_data", braintreeData.toString())
+                .appendQueryParameter("braintree_sdk_data", Base64.encodeToString(braintreeData.toString().getBytes(), 0))
```

Comment:
We could a test for the URL param to make sure the JSON is encoding properly to Base64.

Reply:
That's a good call, added here: https://github.com/braintree/braintree_android/pull/945/commits/3e6fd0568be4272dd925d45a193c7a40d6b0651c